src/screens/CreatePost/elements/PollItem.js

```diff
@@ -66,15 +66,15 @@
     paddingHorizontal: dimen.normalizeDimen(8)
   },
 
-  focuspollitemcontainer: {
+  focuspollitemcontainer: (isAnonym) => ({
     display: 'flex',
     flexDirection: 'row',
     borderWidth: 1,
     borderRadius: 10,
     marginVertical: dimen.normalizeDimen(4),
     paddingHorizontal: dimen.normalizeDimen(8),
-    borderColor: COLORS.signed_primary
-  },
+    borderColor: isAnonym ? COLORS.anon_primary : COLORS.signed_primary
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes made here are good as they allow the border color of the focused poll item container to change based on whether the user is anonymous or not. This can help improve the user experience by providing visual feedback that corresponds with their anonymity status.


<!-- This is an auto-generated comment by OSS CodeRabbit -->